MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -0,0 +1,194 @@
+import Foundation
+import MapboxDirections
+import MapboxNavigationNative
+
+public typealias OfflineDirectionsCompletionHandler = (_ numberOfTiles: UInt64) -> Void
```

Comment:
This type alias appears to be part of a public API, not merely exposed publicly so that it can be used in MapboxNavigation. It should be documented so that the protocol methods using it don’t end up with an enigmatic argument.